Core/src/main/java/org/tribuo/ensemble/WeightedEnsembleModel.java

```diff
@@ -112,4 +134,78 @@ public WeightedEnsembleModel(String name, EnsembleModelProvenance description, I
     protected EnsembleModel<T> copy(String name, EnsembleModelProvenance newProvenance, List<Model<T>> newModels) {
         return new WeightedEnsembleModel<>(name,newProvenance,featureIDMap,outputIDInfo,newModels,combiner);
     }
+
+    /**
+     * Creates an ensemble from existing models. The model outputs are combined using uniform weights.
+     * <p>
+     * Uses the feature and output domain from the first model.
+     * <p>
+     * If the output domains don't cover the same dimensions then it throws {@link IllegalArgumentException}.
+     * @param name The ensemble name.
+     * @param models The ensemble members.
+     * @param combiner The combination function.
+     * @param <T> The output type.
+     * @return A weighted ensemble model.
+     */
+    public static <T extends Output<T>> WeightedEnsembleModel<T> createEnsembleFromExistingModels(String name, List<Model<T>> models, EnsembleCombiner<T> combiner) {
+        return createEnsembleFromExistingModels(name,models,combiner,Util.generateUniformVector(models.size(), 1.0f/models.size()));
+    }
+
+    /**
+     * Creates an ensemble from existing models. The model outputs are combined using uniform weights.
```

Comment:
```suggestion
     * Creates an ensemble from existing models.
```

Presumably this is just copy-pasted from the above docs.

Reply:
Yep, thanks.